### PR TITLE
Improved logging control

### PR DESCRIPTION
**Pull request by PouriaRouzrokh**
Hi Bardia! 

Please see the commits below and the code changes. I will send you a voice message with a detailed explanation shortly.

Thanks,
Pouria

### CODE REVIEW

File: mediffusion/default_config/default.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ validation:
   classifier_cond_scale: 4
   protocol: DDIM100
   log_original: false
+  log_concat: false
+  cls_log_indices: -1 # log the entire cls vector; alternatively, provide a list of desired indices to log
```
Comment (BardiaKh) [inline mediffusion/default_config/default.yaml]: Could you please remove the comment in the file and add it to the table in the readme?

```suggestion
  cls_log_indices: -1
```

File: mediffusion/ddpm.py [hunk 2]
```diff
@@ -161,6 +161,7 @@ def training_step(self, batch, batch_idx):
     def validation_step(self, batch, batch_idx):
         x_start = batch["img"]
         cls = batch["cls"] if self.class_conditioned else None
+        cls_log_indices = self.config.validation.cls_log_indices
```
Comment (BardiaKh) [inline mediffusion/ddpm.py]: I suggest moving `cls_log_indices = self.config.validation.cls_log_indices` to the _log_image method as it is only going to be used there. We can then remove the `cls_log_indices` argument for that as it is going to be read from the config directly. What do you think?

**Review by BardiaKh — COMMENTED** [review]
This is amazing, Pouria! Thanks a lot. I put a couple of comments on the PR. I also pushed an update for the package setup and versions to fix what we discussed. If you think the new changes are okay, you can remove the changes to the setup and requiremnts in this PR.

~BKh

**Comment by PouriaRouzrokh** [comment]
Bardia, I addressed your comments and updated the PR. Can you kindly check if everything is fixed?

**Pull request merged**